fossunited/chapters/doctype/foss_chapter_event/foss_chapter_event.py

```
@@ -36,27 +36,23 @@ class FOSSChapterEvent(WebsiteGenerator):
     if TYPE_CHECKING:
         from frappe.types import DF
 
-        from fossunited.chapters.doctype.foss_chapter_event_member.foss_chapter_event_member import (  # noqa: E501
+        from fossunited.chapters.doctype.foss_chapter_event_member.foss_chapter_event_member import (  # noqa: E501 # noqa: E501
```

Ruff (0.14.7)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)